nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors/src/main/java/org/apache/nifi/processors/splunk/QuerySplunkIndexingStatus.java

```diff
@@ -194,7 +197,11 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 final EventIndexStatusResponse splunkResponse = unmarshallResult(responseMessage.getContent(), EventIndexStatusResponse.class);
 
                 splunkResponse.getAcks().entrySet().forEach(result -> {
-                    final FlowFile toTransfer = undetermined.get(result.getKey());
+                    FlowFile toTransfer = undetermined.get(result.getKey());
```

Comment:
I would go with an approach which keeps final in place. Like: introduce a helper method, which extracts and returns the toTransfer file from the undetermined map and enriches it with the attribute internally

Reply:
Thanks, added back final